oauthenticator/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .oauth2 import *
 from .github import *
 from .bitbucket import *
+from .mediawiki import *
```

Comment:
since mediawiki has added dependencies, lets exclude it from `__init__` here. I only put the ones that existed before the multi-module split here, and don't think we should add more.